lib/samples/query_table_statistics/query_table_statistics_sample.dart

```diff
@@ -29,114 +31,210 @@ class QueryTableStatisticsSample extends StatefulWidget {
 
 class _QueryTableStatisticsSampleState extends State<QueryTableStatisticsSample>
     with SampleStateSupport {
+  // Create a controller for the map view.
   final _mapViewController = ArcGISMapView.createController();
-  final _serviceFeatureTable = ServiceFeatureTable.withUri(Uri.parse(
-      'https://sampleserver6.arcgisonline.com/arcgis/rest/services/SampleWorldCities/MapServer/0'));
-  bool _onlyCitiesInCurrentExtent = true;
-  bool _onlyCitiesGreaterThan5M = true;
-  final _statisticDefinitions = List<StatisticDefinition>.empty(growable: true);
-
-  @override
-  void initState() {
-    super.initState();
-
-    for (final type in StatisticType.values) {
-      _statisticDefinitions.add(
-        StatisticDefinition(onFieldName: 'POP', statisticType: type),
-      );
-    }
-
-    final map = ArcGISMap.withBasemapStyle(BasemapStyle.arcGISTopographic);
-    final featureLayer = FeatureLayer.withFeatureTable(_serviceFeatureTable);
-    map.operationalLayers.add(featureLayer);
-
-    _mapViewController.arcGISMap = map;
-  }
+  // Create a ServiceFeatureTable from a URL.
+  final _serviceFeatureTable = ServiceFeatureTable.withUri(
+    Uri.parse(
+      'https://sampleserver6.arcgisonline.com/arcgis/rest/services/SampleWorldCities/MapServer/0',
+    ),
+  );
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+  // A flag to limit the query to the current display extent..
+  var _onlyCitiesInCurrentExtent = true;
+  // A flag for whether to limit the query to cities with population greater than 5 million.
+  var _onlyCitiesGreaterThan5M = true;
+  // A flag for whether to limit the query to cities within the current extent.
```

Comment:
This comment is in the wrong place -- it should be up a few lines, above `_onlyCitiesInCurrentExtent`